pkg/build/builder/cmd/builder.go

```diff
@@ -242,3 +295,30 @@ func RunDockerBuild(out io.Writer) error {
 func RunS2IBuild(out io.Writer) error {
 	return runBuild(out, s2iBuilder{})
 }
+
+// RunGitClone performs a git clone using the build defined in the environment
+func RunGitClone(out io.Writer) error {
+	cfg, err := newBuilderConfigFromEnvironment(out, false)
+	if err != nil {
+		return err
+	}
+	return cfg.clone()
+}
+
+// RunManageDockerfile manipulates the dockerfile for docker builds
```

Comment:
well it's not really just rewrite either, it's:

1) if you provided an inline dockerfile, write it to disk (possibly replacing something from git or the binary inputs, or possibly just being a clean write)
2) check if a dockerfile exists in the contextdir/dockerfilepath specified (including from (1))
3) update that dockerfile (update FROM, add env, add labels)

that's why i went with "manage".  rewrite doesn't seem to cover it to me.  Happy to add a more descriptive comment though (or another naming suggestion).